lib/services/debug-service.ts

```diff
@@ -35,9 +37,16 @@ class DebugService extends EventEmitter {
 		// After we find a way to check on iOS we should use it here.
 		const isAppRunning = true;
 		let result: string[];
-		debugOptions.chrome = !debugOptions.client;
+		debugOptions.chrome = true;
```

Comment:
Why we want to set the `debugOptions.chrome` to true here? If we keep the old value we will default to chrome debugging but we will keep the app inspector debugging option.